src/sources/mssql/mssql-schema.lisp

```diff
@@ -4,48 +4,26 @@
 
 (in-package :pgloader.source.mssql)
 
-(defvar *mssql-db* nil
-  "The MS SQL database connection handler.")
-
-;;;
-;;; General utility to manage MySQL connection
-;;;
-(defclass mssql-connection (db-connection) ())
-
-(defmethod initialize-instance :after ((msconn mssql-connection) &key)
-  "Assign the type slot to mssql."
-  (setf (slot-value msconn 'type) "mssql"))
-
-(defmethod open-connection ((msconn mssql-connection) &key)
-  (setf (conn-handle msconn) (mssql:connect (db-name msconn)
-                                            (db-user msconn)
-                                            (db-pass msconn)
-                                            (db-host msconn)))
-  ;; apply mysql-settings, if any
-  (loop :for (name . value) :in *mssql-settings*
-     :for sql := (format nil "set ~a ~a;" name value)
-     :do (query msconn sql))
-
-  ;; return the connection object
-  msconn)
-
-(defmethod close-connection ((msconn mssql-connection))
-  (mssql:disconnect (conn-handle msconn))
-  (setf (conn-handle msconn) nil)
-  msconn)
-
-(defmethod clone-connection ((c mssql-connection))
-  (change-class (call-next-method c) 'mssql-connection))
-
-(defmethod query ((msconn mssql-connection) sql &key)
-  "Send SQL query to MSCONN connection."
-  (log-message :sql "MSSQL: sending query: ~a" sql)
-  (mssql:query sql :connection (conn-handle msconn)))
-
-(defun mssql-query (query)
-  "Execute given QUERY within the current *connection*, and set proper
-   defaults for pgloader."
-  (query *mssql-db* query))
+(defclass copy-mssql (db-copy)
+  ((encoding :accessor encoding         ; allows forcing encoding
+             :initarg :encoding
+             :initform nil))
+  (:documentation "pgloader MS SQL Data Source"))
+
+(defmethod initialize-instance :after ((source copy-mssql) &key)
+  "Add a default value for transforms in case it's not been provided."
+  (let* ((transforms (when (slot-boundp source 'transforms)
```

Comment:
Same issue as the one I've mentioned below.

Reply:
Fixed.